six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -199,6 +221,17 @@ void writeNITF(const std::string& pathname, const std::vector<std::string>&
 Data* readNITF(const std::string& pathname,
         const std::vector<std::string>& schemaPaths);
 
+void crop_SICD(const std::string& inPathname,
+               const std::vector<std::string>& schemaPaths,
+               const types::RowCol<size_t>& aoiOffset,
+               const types::RowCol<size_t>& aoiDims,
+               const std::string& outPathname);
+
+six::sicd::ComplexData* const cropMetaData(
```

Comment:
Why is this here?  Doesn't seem to do anything..?

Is this to get Swig to generate the function?  If you do a `#include <six/sicd/CropUtils.h>` on top and a `%include <six/sicd/CropUtils.h>` farther down, I don't think you need these... Swig should just do it.

@JonathanMeans can help if you're having trouble getting Swig to generate what you want.

Reply:
As mentioned in comment above I don't want swig to wrap all the version of cropSICD in CropUtils.h since those can't be called directly.